### PR TITLE
Update metadata.authors & USAGE

### PR DESCRIPTION
**Pull request by dhimmel**
metadata.yaml should now use `authors` instead of `author_info`.

Update USAGE to mention delete-me.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -179,7 +182,7 @@ The following YAML shows the supported key–value pairs for an author:
 ```yaml
 github: dhimmel  # strongly suggested
 name: Daniel S. Himmelstein  # mandatory
-initials: DSH  # strongly suggested
+initials: DSH  # optional
```
Comment (dhimmel) [inline USAGE.md]: initials are not used anywhere by default, so seems excessive to call them "strongly suggested"

File: USAGE.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ Markdown files are identified by their `.md` extension and ordered according to
 For basic formatting, check out the [CommonMark Help](https://commonmark.org/help/) page for an introduction to the formatting options provided by standard markdown.
 In addition, Manubot supports an extended version of markdown, tailored for scholarly writing, which includes [Pandoc's Markdown](https://pandoc.org/MANUAL.html#pandocs-markdown) and the extensions discussed below.
 
+The `content/02.delete-me.md` file in the Rootstock repository shows many of the elements and formatting options supported by Manubot.
+See the [raw markdown](https://gitlab.com/manubot/rootstock/blob/master/content/02.delete-me.md#L) in this file and compare it to the [rendered manuscript](https://manubot.github.io/rootstock/).
```
Comment (dhimmel) [inline USAGE.md]: Meant to help address https://github.com/manubot/rootstock/issues/288#issuecomment-567065243.

CC @Nebucatnetzer and @vincerubinetti

Comment (dhimmel) [reply USAGE.md]: Using GitLab link to raw markdown as a workaround for https://github.com/isaacs/github/issues/297. Will be slightly confusing, but not sure what is best.

File: USAGE.md (hunk 1 above)
Comment (agitter) [reply USAGE.md]: Agreed.  I think that was a remnant from deep review where we were trying to automate things like funding statements.  We haven't needed them in more modern forms of Manubot.

**Pull request merged**